src/dbnode/storage/index/types.go

```diff
@@ -273,6 +290,16 @@ type Block interface {
 		results BaseResults,
 	) (exhaustive bool, err error)
 
+	// Aggregate aggregates known tag names/values.
+	// NB(prateek): this is different from Query, as we can
```

Comment:
Nah, methods on this interface are `Query()` and `Aggregate()`. Unless you mean to say rename?

Reply:
`different from aggregating by means of Query` maybe? Otherwise it kinda reads like this aggregates tag names/values which you don't really expect Query to do

Reply:
lol sure